README.md

```diff
@@ -25,29 +25,22 @@
 
 <div align="center">
 
-[![Crate Badge]](https://crates.io/crates/ratatui)
-[![License Badge]](./LICENSE)
-[![CI Badge]](https://github.com/ratatui-org/ratatui/actions?query=workflow%3ACI+)
-[![Docs Badge]](https://docs.rs/crate/ratatui/)<br>
-[![Dependencies Badge]](https://deps.rs/repo/github/ratatui-org/ratatui)
-[![Codecov Badge]](https://app.codecov.io/gh/ratatui-org/ratatui)
-[![Discord Badge]](https://discord.gg/pMCEU9hNEj)
-[![Matrix Badge]](https://matrix.to/#/#ratatui:matrix.org)<br>
-
-[Documentation](https://docs.rs/ratatui)
-· [Ratatui Website](https://ratatui.rs)
-· [Examples](https://github.com/ratatui-org/ratatui/tree/main/examples)
-· [Report a bug](https://github.com/ratatui-org/ratatui/issues/new?labels=bug&projects=&template=bug_report.md)
-· [Request a Feature](https://github.com/ratatui-org/ratatui/issues/new?labels=enhancement&projects=&template=feature_request.md)
-· [Send a Pull Request](https://github.com/ratatui-org/ratatui/compare)
+[![Crate Badge]][Crate] [![Docs Badge]][API Docs] [![CI Badge]][CI Workflow] [![License
+Badge]](./LICENSE)<br>
+[![Codecov Badge]][Codecov] [![Deps.rs Badge]][Deps.rs] [![Discord Badge]][Discord Server]
+[![Matrix Badge]][Matrix]<br>
+
+[Ratatui Website] · [API Docs] · [Examples]<br>
+[Contributing] · [Changelog] · [Breaking Changes]<br>
```

Comment:
Can we move `[Contributing]` to the next line?

First line for new users
Second line for users hoping to upgrade
Third line for contributors